libs/platform/src/lib/components/button/button.component.ts

```diff
@@ -7,9 +7,16 @@ import {
     ElementRef
 } from '@angular/core';
 
-export type ButtonType = 'standard' | 'positive' | 'medium' | 'negative' | 'toolbar' | 'main';
-export type ButtonOptions = 'light' | 'emphasized' | '';
-
+export type ButtonType =
+    | ''
```

Comment:
Why do we need to redefine ButtonType, why we cannot use it? I see some new option  but what are those, do you have functionality or styling for it?

Reply:
Styles are defined for these types.
and the changes are 

1).Light is renamed to Transparent.
2).Half type is removed.
3).emphasized will be part of type.
4).button options won't be supported, so as per 0.7 style update added emphasized, ghost, transparent to button types.

Reply:
Are you saying that core does not have this definition ? It does not have to perfect and they dont need to have everything, but let;s use one Type. . 

I assume for this you need to have support in styles as well right ?